src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -63,6 +63,26 @@ public function load($relations)
         return $this;
     }
 
+    public function loadCount($relations)
+    {
+        if ($this->isEmpty()) {
+            return $this;
+        }
+
+        $relations = is_array($relations) ? $relations : func_get_args();
+
+        $query = $this->first()->newQueryWithoutRelationships()
+            ->whereKey($this->modelKeys())
+            ->select($this->first()->getKeyName())
+            ->withCount($relations);
```

Comment:
You can use `->withCount(...func_get_args())` and remove line 72.

Reply:
Noice one, much nicer.